src/components/Sidebar/Sidebar.js

```diff
@@ -203,6 +203,7 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = {
   createNewProjectStart: actions.createNewProjectStart,
+  selectProject: actions.selectProject,
```

Comment:
Looks like this was accidentally removed in https://github.com/joshwcomeau/guppy/commit/4cd03213a572f33409abe0d035ba7322cf049b97#diff-ea3d101cb36d8e83a8294a5a4a6bba82L206

Reply:
Nice catch, this fixes the error message I noted in #176.